app/views/layouts/_discussions_list.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="discussions">
   <% @filtered_discussions.each do |discussion| %>
-    <%= link_to exercise_discussion_path(discussion.exercise.id, discussion) do %>
+    <%= link_to exercise_discussion_path(discussion.exercise.id, discussion), class: 'discussion-link' do %>
```

Comment:
I wrote this on the commit message too, but here it goes again:

Most organizations change the link colors through custom CSS.
Since each discussion on the index is wrapped in an `<a>` tag,
all text would change to the link color, which doesn't look very
good. Now that the tag can be easily identified, link colors can
be changed using `a:not(.discussion-link)`.